app/utils/answersHelpers/consignment/DocumentAnswersHelper.scala

```diff
@@ -84,4 +84,11 @@ class DocumentAnswersHelper(
         call = Some(Call(GET, "#")) //TODO change me please
       )
     }
+
+  val documentAddRemoveLink: Link = Link(
+    id = s"add-remove-documents-${documentIndex.display}",
```

Comment:
Does it make sense to have the document index in the id? This is a link for the entire documents section, not an individual document. Feel like this might be in the wrong class. This applies across the board.

Reply:
So the reason for this is that if we have 3 documents on the cross-check page we will have 3 links (all doing the same thing) and I was concerned that if the document index wasn't included we would have 3 duplicate ID's? I may be missing something but that was the rationale behind it - The only one which isn't following this pattern is AdditionalReferences where we only have a single Accordion so that doesn't have an index in it

Reply:
Good point, looking at the prototype its showing the links within the accordion, meaning we would have a link for each transport equipment for example, but this doesn't seem right. In which case I don't think we need the id here, and also a bit more refactoring is needed 😁